velox/core/PlanNode.h

```diff
@@ -764,6 +764,56 @@ class TableWriteMergeNode : public PlanNode {
   const RowTypePtr outputType_;
 };
 
+/// Plan node used to apply all of the projections expressions to every input
```

Comment:
This documentation is not clear and needs updating. Here is one way to structure this comment.

```
For each input row, generates N rows with M additional columns according to specified 'projections'.

'Projections' is an N x M matrix of expressions. Each expression is either a column reference or a constant. Both null and non-null constants are allowed. 

'Names' is a list of new column names. It contains exactly M entries.

The semantic of this operator matches Spark.
```

Let's remove a restriction that constant expression's type must be integer. It seems redundant.